### PR TITLE
Fix some incorrect types

### PR DESCRIPTION
**Pull request by vonElfvin**

### CODE REVIEW

File: src/components/claims/claim-details/components/inventory/ClaimInventory.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ export class ClaimInventory extends React.Component {
               claimId={this.props.claimId}
             />
             <Button
-              size="tiny"
+              size="small"
```
Comment (palmenhq) [inline src/components/claims/claim-details/components/inventory/ClaimInventory.js]: this button is not a `hedvig-ui/button` button though :)

File: src/components/tickets/ticket/history/revision.tsx [hunk 2]
```diff
@@ -32,7 +32,7 @@ export class TicketRevision extends React.Component<ITicketRevision, {}> {
         </Label>
         <Button
           compact
-          size="tiny"
+          size="small"
```
Comment (palmenhq) [inline src/components/tickets/ticket/history/revision.tsx]: same here, this comes from `semantic-ui-react` not `hedvig-ui`, and `semantic-ui-react` is fine with "tiny" buttons

File: src/components/chat/tabs/insurance-tab/create-quote.tsx [hunk 3]
```diff
@@ -97,7 +97,7 @@ export const CreateQuote: React.FunctionComponent<{
         )
       }
       open={modalOpen}
-      size="tiny"
+      size="small"
```
Comment (palmenhq) [inline src/components/chat/tabs/insurance-tab/create-quote.tsx]: i think this button is safe also :)

File: src/components/chat/tabs/quotes/quote-modification.tsx [hunk 4]
```diff
@@ -421,7 +421,7 @@ const ExtraBuildingEditor: React.FC<{
           <RemoveButtonWrapper>
             <Button
               variation="danger"
-              size="tiny"
+              size="small"
```
Comment (palmenhq) [inline src/components/chat/tabs/quotes/quote-modification.tsx]: THIS one is really nasty though, nice find!

**Comment by vonElfvin** [comment]
Fixed by #144 other PR

**Pull request closed without merging**